src/config/env.ts

```diff
@@ -1,6 +1,25 @@
 // load variable from .env file, only used in local dev env
 import "dotenv/config";
 
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
```

Comment:
Not sure what this does, can you explain the change ?

Reply:
It's a common error when activating the option.
As Typescript has not idea what properties you use on `process.env.*`
https://stackoverflow.com/questions/45194598/using-process-env-in-typescript
better bypass the typescripts errors (the `process.env.MOULA!`) ?